app/lib/nomis_client/prisoner_category.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module NomisClient
+  class PrisonerCategory
```

Comment:
Since this class is querying the generic bookings endpoint, maybe we should avoid naming it `PrisonCatgeory` and go with something more general incase we extend it to pull more attributes later?

Reply:
hmm, yes I ummmmed and arrrred over that, but happy to rename to something more generic (`booking_details`?) if that is the consensus view